.golangci.yml

```diff
@@ -10,6 +10,9 @@ linters-settings:
     list-type: blacklist
     packages:
       - github.com/FerretDB/FerretDB/internal/bson
+      - github.com/FerretDB/FerretDB/internal/fjson
+      - github.com/FerretDB/FerretDB/internal/tjson
+      - github.com/FerretDB/FerretDB/internal/handlers/pg/pgdb
```

Comment:
why? When to uncomment?

Reply:
> Why?

Other changes and comments in that file should make it clear

> When to uncomment?

It is not commented out